test/utils/JsonYamlConverterSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import io.circe.yaml.syntax.AsYaml
+import testUtils.TestSupport
+
+import scala.io.Source
+
+class JsonYamlConverterSpec extends TestSupport {
+
+  val converter = new JsonYamlConverter()
+
+  // Helper method to read file content
+  def readFile(path: String): String = {
+    val source = Source.fromFile(path)
+    try {
+      source.getLines().mkString("\n")
+    } finally {
+      source.close()
+    }
+  }
+
+  def formatJson(jsonString: String): String = {
```

Comment:
basically testing the library a bit but it's a pain comparing yaml strings and json strings hence the formatting

Reply:
decided to compare the parsing of files rather than a hard coded string, basically the indentation and formatting was a nightmare to compare.